docs/rfcs/9999-vpn-firefox-integration.md

```diff
@@ -0,0 +1,167 @@
+- Status: Draft
+- Date: 2023-05-29
+- Authors:
+    - [@strseb](https://github.com/strseb)
+    - [@lesleyjanenorton](https://github.com/lesleyjanenorton)
+    - [@oskirby](https://github.com/oskirby)
+- RFC PR: -
+- Implementation GitHub issue: -
+- [Design @Figma](https://www.figma.com/file/s13B7zs27cadZXUyvxobgW/Better-Together-Explorations?type=design&node-id=259-4830&mode=design&t=4BmtgXMzjnubd9gc-0)
+
+-----
+# Firefox + VPN Better together

```

Comment:
I have a random question: if we are able to do split tunneling per URL domain, why don't we provide that feature in the app as well as in the extension?

Reply:
Ah ok I think I understand that we actually can't do this, this is only possible due to the Proxy browser extension stuff.